nat-handler.py

```diff
@@ -94,17 +102,30 @@ def current_az(self):
     def current_region(self):
         return self.current_az[:-1]
 
+    @property
+    def primary_interface_id(self):
+        ec2 = connect_to_ec2(self.current_region)
+        instance = ec2.get_only_instances(instance_ids=[self.current_instance_id])[0]
+        interfaces = instance.interfaces
+        interface_id = [i.id for i in interfaces if i.id != self.secondary_interface_id]
+        return interface_id[0]
+
+    @property
+    def secondary_interface_id(self):
+        return self.config.secondary_interface_id(self.current_az)
+
     def take_route(self, az):
         route_table_id = self.config.route_table_id(az)
         vpc = connect_to_vpc(self.current_region)
-        vpc.replace_route(route_table_id, '0.0.0.0/0', instance_id=self.current_instance_id)
+        vpc.replace_route(route_table_id, '0.0.0.0/0', interface_id=self.primary_interface_id)
 
     def take_elastic_ip(self, az):
+        interface_id = self.primary_interface_id if az == self.current_az else self.secondary_interface_id
```

Comment:
I thought of updating the readme, but the readme talked about failover in general. At the present scenario, it did not say how failover is achieved but rather that it only spoke about the taking over of routes. So i thought it was not necessary .

But not to worry, i will gladly update the readme :)